docs/adr/0003-allowing-implicit-object-type.md

```diff
@@ -0,0 +1,47 @@
+# Allowing implicit object type
+
+## Status
+
+Proposed.
+
+## Context
+
+In real-world OpenAPI schema, we have discovered that people frequently omit the `type: object` from their object definitions. THIS IS NOT VALID Open API schema.
+
+```
```

Comment:
Consider qualifying this, e.g.:

```
```yaml
```

When you do that, yaml in GitHub gets a modicum of syntax colouring:

```yaml
Pet:
    required:
    - id
    - name
    properties:
    id:
        type: integer
        format: int64
```

Reply:
Done!